config/initializers/logging.rb

```diff
@@ -0,0 +1,20 @@
+config = Rails.application.config
+
+if ENV["RAILS_LOG_TO_STDOUT"].present?
+  # heroku and docker: dump everything to stdout
+  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+elsif ENV["RAILS_LOG_TO_SYSLOG"]
```

Comment:
yeah good point, saw that a few times too ... I'll add a boolean env reader
or so ...

On Wed, Jun 29, 2016 at 12:12 PM, Jon Moter <notifications@github.com>
wrote:

> In config/initializers/logging.rb
> <https://github.com/zendesk/samson/pull/1114#discussion_r69009362>:
>
> > @@ -0,0 +1,20 @@
> > +config = Rails.application.config
> > +
> > +if ENV["RAILS_LOG_TO_STDOUT"].present?
> > +  # heroku and docker: dump everything to stdout
> > +  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
> > +elsif ENV["RAILS_LOG_TO_SYSLOG"]
>
> I want to be consistent in how we handle boolean environment variables. In
> our .env.example we have some values that are 1 or 0, some that have values
> like "true", and apparently some that are enabled if the environment
> variable is set at all. That's confusing.
>
> This is outside the scope of this PR, but I'm bringing it up so that we
> can try to get some consistency.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1114/files/aaba7c5c0b927f0a41dc8943a040e50c77b46577#r69009362>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ7n1QoeudCmAG4N9enQnTH6UCHxJks5qQsOzgaJpZM4JAxhu>
> .
>